extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -88,6 +84,13 @@ const TEMPLATE = {
       tag: 'div',
       attrs: dict({'class': 'i-amphtml-story-system-layer-buttons'}),
       children: [
+        {
```

Comment:
This would break for a few cases:
- RTL
- First page where the (i) icon is visible

Can we use the same message container that's already implemented, instead of adding a new one? Making it generic looks trivial, I think renaming `.i-amphtml-story-sound-display` should be enough. Then you'd get RTL and (i) icon support out of the box.

Reply:
Is the UX the same?  I think the audio message "comes out of" the mute/unmute icon.  Is that the same for this?

Reply:
Asking @hongwei1990 , I'll update the PR when I get the details.

Reply:
Technically the text is displayed by the buttons. The audio icon happens to be the first one, so we get that effect, but if there's the (i) icon we wouldn't
It's Hong's call, but from what I see in this PR it should be fine :)